extensions/dbt-documentation/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { ActionPanel, getPreferenceValues, List, Action, showToast, Toast } from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliasearch from "algoliasearch/lite";
+
+const APPID = "57ZWAOQC7F";
+const APIKEY = "04a5092253f5120fdff2c77b3847d0e1";
```

Comment:
why didn't you consider adding this as a preference? I guess this is not an account or user related API key?

Reply:
These are the API creds to connect to Algolia for it to return results for the dbt documentation. This is not something that the users need to know/fill-in themselves.

This is the same approach used by the vuejs-documentation extension:
https://github.com/raycast/extensions/blob/0a576d1ce4a411840d8ff1765c0ab3d67d999708/extensions/vuejs-documentation/src/index.tsx#L8-L10